percy/providers/appAutomateProvider.js

```diff
@@ -33,10 +33,12 @@ class AppAutomateProvider extends GenericProvider {
     scrollableXpath,
     topScrollviewOffset,
     bottomScrollviewOffset,
-    scrollableId
+    scrollableId,
+    sync
   } = {}) {
     let response = null;
     let error;
+    sync = sync || false;
```

Comment:
Remove default value

Reply:
for app automate we can have default value right? Removing it and adding null.